src/main/fc/fc_msp.c

```diff
@@ -2145,8 +2144,31 @@ void mspFcProcessReply(mspPacket_t *reply)
     UNUSED(src); // potentially unused depending on compile options.
 
     switch (reply->cmd) {
-        case MSP_DISPLAYPORT: {
+#ifndef OSD_SLAVE
+    case MSP_ANALOG:
+        {
```

Comment:
@ledvinap , we had a big discussion on how switch statements should be formatted. After much discussion this was the agreed format when switch statements declared local variables.